### PR TITLE
Cloud9 Support for Stacksets

### PR DESCRIPTION
**Pull request by AvinashSeelam9**
## What does this PR do and why?
This PR provides Cloud9 support for StackSets
_Describe in detail what your merge request does and why._
This PR provides Cloud9 support for StackSets
## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/intermediate/operations/stacksets/index.md [hunk 1]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    
+    **How to delete AWS CloudFormation stacks within stack set**
+    
+    1. Navigate to the [AWS CloudFormation StackSets console.](https://console.aws.amazon.com/cloudformation/home#/stacksets)
+    1. Select the CloudFormation stack set you want to delete the stacks from. Choose the last stack set you created, i.e., `cfn-workshop-ec2instance`.
+    1. From top-right section of the page, select **Actions**, and choose **Delete stacks from StackSet**.
+    1. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.
+    1. Under **Account numbers** enter the 12-digit AWS account ID for the account you are using for this lab.
+    1. For **Specify regions** select **Add all regions**. This will automatically select the AWS Regions that the StackSet deployed stacks into. Choose **Next**.
+    1. The **Status** changes to `PENDING`.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:320]:
```suggestion
    1. On the **Review** page, review the contents of the page, and choose **Submit**.
    1. The **Status** changes to `PENDING`.
```

File: content/intermediate/operations/stacksets/index.md [hunk 2]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    Now that you have deleted stacks within each StackSet, you will now choose to delete the empty StackSet.
+    
+    **How to delete an AWS CloudFormation stack set**
+    
+    
+    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).
+    2. Select the stack set you wish to delete.
+    3. Choose **Actions**, and then **Delete StackSet**.
+    4. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.
+    5. In the popup that appears, confirm you want to delete this stack set by choosing **Delete StackSet**.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:333]:
```suggestion
    1. In the popup that appears, confirm you want to delete this stack set by choosing **Delete**.
```

File: content/intermediate/operations/stacksets/index.md [hunk 3]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+
+    Now that you have deleted stacks within each StackSet, you will now choose to delete the empty StackSet.
+    
+    **How to delete an AWS CloudFormation stack set**
+    
+    
+    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).
+    2. Select the stack set you wish to delete.
+    3. Choose **Actions**, and then **Delete StackSet**.
+    4. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:332]:
```suggestion
```

File: content/intermediate/operations/stacksets/index.md [hunk 4]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    5. In the popup that appears, confirm you want to delete this stack set by choosing **Delete StackSet**.
+    6. On refresh, your StackSet should no longer be listed.
+    7. Follow steps 2 through 6 for the two other stack sets.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:335]:
```suggestion
    1. Follow steps 2 through 5 for the two other stack sets.
```

File: content/intermediate/operations/stacksets/index.md [hunk 5]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    
+    **How to delete an AWS CloudFormation stack set**
+    
+    
+    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).
+    2. Select the stack set you wish to delete.
+    3. Choose **Actions**, and then **Delete StackSet**.
+    4. Under **Accounts**, select **Deploy stacks in accounts** under **Deployment locations**.
+    5. In the popup that appears, confirm you want to delete this stack set by choosing **Delete StackSet**.
+    6. On refresh, your StackSet should no longer be listed.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:334]:
```suggestion
    1. On refresh, your StackSet should no longer be listed.
```

File: content/intermediate/operations/stacksets/index.md [hunk 6]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    1. Follow steps 2 through 8 for the other two stack sets, and in the following order: `cfn-workshop-security`, and `cfn-workshop-network`.
+
+    Now that you have deleted stacks within each StackSet, you will now choose to delete the empty StackSet.
+    
+    **How to delete an AWS CloudFormation stack set**
+    
+    
+    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).
+    2. Select the stack set you wish to delete.
+    3. Choose **Actions**, and then **Delete StackSet**.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:331]:
```suggestion
    1. Choose **Actions**, and then **Delete StackSet**.
```

File: content/intermediate/operations/stacksets/index.md [hunk 7]
```diff
@@ -184,37 +263,78 @@ In this exercise, you will use the knowledge gained from earlier parts of this l
...
+    1. Refresh until the **Status** changes to `SUCCEEDED`.
+    1. Follow steps 2 through 8 for the other two stack sets, and in the following order: `cfn-workshop-security`, and `cfn-workshop-network`.
+
+    Now that you have deleted stacks within each StackSet, you will now choose to delete the empty StackSet.
+    
+    **How to delete an AWS CloudFormation stack set**
+    
+    
+    1. Navigate to [AWS CloudFormation StackSets console](https://console.aws.amazon.com/cloudformation/home#/stacksets).
+    2. Select the stack set you wish to delete.
```
Comment (cjhar) [inline content/intermediate/operations/stacksets/index.md:330]:
```suggestion
    1. Select the stack set you wish to delete.
```

**Review by cjhar — CHANGES_REQUESTED** [review]
Thanks please see review comments and also fix

line 59 8. Select **Submit**.

**Review by cjhar — DISMISSED** [review]
LGTM

**Review by rezabekf — APPROVED** [review]
Approved.

**Pull request merged**